pkg/cloud/metadata/k8s.go

```diff
@@ -18,7 +23,39 @@ var DefaultKubernetesAPIClient = func() (kubernetes.Interface, error) {
 	// creates the in-cluster config
 	config, err := rest.InClusterConfig()
 	if err != nil {
-		return nil, err
+		if strings.Contains(err.Error(), "The system cannot find the path specified") {
```

Comment:
Is there an explicit error we can check for with [errors.Is](https://pkg.go.dev/errors#Is) instead of using `strings.Contains`?

Reply:
Unfortunately, no: https://github.com/kubernetes/client-go/blob/3aa45779f2e5592d52edf68da66abfbd0805e413/rest/config.go#L523

Reply:
Updated: 
```
kubectl logs -f ebs-csi-node-windows-mlb7k -n kube-system                               
Defaulted container "ebs-plugin" out of: ebs-plugin, node-driver-registrar, liveness-probe
I0425 20:52:50.011887    5376 ec2.go:40] "Retrieving EC2 instance identity metadata" regionFromSession=""
I0425 20:52:50.064112    5376 metadata.go:48] "ec2 metadata is available"
I0425 20:52:50.065664    5376 k8s.go:27] "InClusterConfig failed to read token file, retrieving file from sandbox mount point"
```